ecl/resource_ecl_mlb_load_balancer_v1.go

```diff
@@ -367,13 +391,16 @@ func resourceMLBLoadBalancerV1UpdateConfigurations(d *schema.ResourceData, clien
 			isConfigurationsUpdated = true
 
 			for i, interfaceV := range d.Get("interfaces").([]interface{}) {
-				results := make([]load_balancers.CreateStagedOptsReservedFixedIP, len(interfaceV.(map[string]interface{})["reserved_fixed_ips"].([]interface{})))
-				for j, reservedFixedIP := range interfaceV.(map[string]interface{})["reserved_fixed_ips"].([]interface{}) {
-					results[j] = load_balancers.CreateStagedOptsReservedFixedIP{
-						IPAddress: reservedFixedIP.(map[string]interface{})["ip_address"].(string),
+				reservedFixedIPs[i] = []load_balancers.CreateStagedOptsReservedFixedIP{}
+				if reservedFixedIPsRaw, ok := interfaceV.(map[string]interface{})["reserved_fixed_ips"].([]interface{}); ok {
+					results := make([]load_balancers.CreateStagedOptsReservedFixedIP, len(reservedFixedIPsRaw))
+					for j, reservedFixedIP := range reservedFixedIPsRaw {
+						results[j] = load_balancers.CreateStagedOptsReservedFixedIP{
+							IPAddress: reservedFixedIP.(map[string]interface{})["ip_address"].(string),
+						}
 					}
+					reservedFixedIPs[i] = results
 				}
```

Comment:
[IMO]
https://github.com/hico-horiuchi/terraform-provider-ecl/pull/1#discussion_r2050456839 の `ipAddress` の対応と合わせて修正して頂いたかと思うのですが、
[resource_ecl_mlb_load_balancer_v1.go#L452-L462](https://github.com/hico-horiuchi/terraform-provider-ecl/blob/32385f2/ecl/resource_ecl_mlb_load_balancer_v1.go#L452-L462) と同様の処理順序の方が分かり易い気がしました

```suggestion
				results := []load_balancers.CreateStagedOptsReservedFixedIP{}
				if reservedFixedIPsRaw, ok := interfaceV.(map[string]interface{})["reserved_fixed_ips"].([]interface{}); ok {
					results = make([]load_balancers.CreateStagedOptsReservedFixedIP, len(reservedFixedIPsRaw))
					for j, reservedFixedIP := range reservedFixedIPsRaw {
						results[j] = load_balancers.CreateStagedOptsReservedFixedIP{
							IPAddress: reservedFixedIP.(map[string]interface{})["ip_address"].(string),
						}
					}
				}
				reservedFixedIPs[i] = results
```

Reply:
処理順序を修正しました